plugin/move.vim

```diff
@@ -82,68 +82,26 @@ function s:MoveBlockVertically(distance)
 
 endfunction
 
-"
-" In normal mode, move the character under the cursor horizontally
-" Moves right (distance > 0) and left if (distance < 0).
-"
-function s:MoveCharHorizontally(distance)
-    if !&modifiable
-        return
-    endif
-
-    let l:curr = virtcol('.')
-    let l:before = l:curr + a:distance
-    if !g:move_past_end_of_line
-        let l:before = max([1, min([l:before, virtcol('$')-1])])
-    endif
-
-    if l:curr == l:before
-        " Don't add an empty change to the undo stack.
-        return
-    endif
-
-    let l:old_default_register = @"
-    let [l:old_virtualedit, &virtualedit] = [&virtualedit, 'all']
-
-    normal! x
-    execute 'normal!' . (l:before.'|')
-    normal! P
-
-    let &virtualedit = l:old_virtualedit
-    let @" = l:old_default_register
-
-endfunction
 
 "
-" In visual mode, move the selected block horizontally.
-" Moves right (distance > 0) and left if (distance < 0).
-" Switches to visual-block mode first if another visual mode is selected.
+" If in normal mode, moves the character under the cursor.
+" If in blockwise visual mode, moves the selected rectangular area.
+" Goes right if (distance > 0) and left if (distance < 0).
+" Returns whether an edit was made.
 "
-function s:MoveBlockHorizontally(distance)
+function s:MoveHorizontally(corner1, corner2, distance)
```

Comment:
Any particular reason to call these parameters `cornerX`? Maybe `from` and `to` instead?

Reply:
Dunno. I think from and to sound like they could refer to the distance that we are moving instead of to the area that we are moving. "Move X from here to there"

What about corner_start and corner_end?

Reply:
Sounds good.

Reply:
OK! It should be fixed in the latest commit now.